docs/publishing/sections/4_analytics_portfolio_site.md

```diff
@@ -9,3 +9,118 @@ For these situations, the [Jupyter Book-based](https://jupyterbook.org/en/stable
 present in the data-analyses repo is your friend.
 
 You can find the Cal-ITP Analytics Portfolio at [analysis.calitp.org](https://analysis.calitp.org).
+
+## Setup:
+Before executing the build, there are a few prior steps you need to do.
+
+1. Set up netlify key:
+    * install netlify: `npm install -g netlify-cli`
+    * Navigate to your main directory
+    * Edit your bash profile:
+        * enter `vi ~/.bash_profile` in the command line
+        * Type `o` to enter a new line
+        * Once in insert mode, can copy and paste the following keys, prefixing it with "export":
+            * `export NETLIFY_AUTH_TOKEN= YOURTOKENHERE123`
+            * `export NETLIFY_SITE_ID=cal-itp-data-analyses`
+        * Press `ESC` + `:wq` to stop insert mode and to close the vim
+            * note: `ESC` + `dd` gets rid of unwanted lines
+        * In the terminal enter command: `env | grep NETLIFY` to see that your Netlify token is there
+
+2. Create a `.yml` file in [data-analyses/portfolio/sites](https://github.com/cal-itp/data-analyses/tree/main/portfolio/sites). Each `.yml` file is a site, so if you have separate research topics, they should each have their own `.yml` file.
+    * This `.yml` file will include the directory to the notebook(s) you want to publish.
+    * Name your `.yml` file
```

Comment:
In this example: `my_report.yml`. That's why to deploy and build, you used `python portfolio/portfolio.py build my_report`.